src/LibChorus/VcsDrivers/Mercurial/HgRepository.cs

```diff
@@ -667,7 +669,7 @@ private string SafeGetStatus()
 		}
 
 		/// <exception cref="System.TimeoutException"/>
-		private static ExecutionResult ExecuteErrorsOk(string command, string fromDirectory, int secondsBeforeTimeout, IProgress progress)
+		private static ExecutionResult ExecuteErrorsOk(string command, string fromDirectory, int secondsBeforeTimeout, IProgress progress, IDictionary<string, ExecutionResult> hgLogCache)
```

Comment:
I think it would be better to make this method non-static instead of passing in the instance variable. As far as I can see all callers are non-static.

Reply:
The person who wrote this really likes static methods, so I was trying not to change the design too much, but I kind of agree. There's nothing that calls this from outside passing in its own Progress instance (it's a private method after all), so I'll make that change.

Reply:
Done.